### PR TITLE
feature: add support for DCI HDR

### PR DESCRIPTION
**Pull request by msheby**
Adds documentation for DCI's HDR.

### CODE REVIEW

File: src/main/data/contentmodifiers.json
Comment (radford-for-smpte) [inline src/main/data/contentmodifiers.json:235]: Let's use the latest HTML version which is under docs.dcimovies.com:

```suggestion
          "url": "https://docs.dcimovies.com/hdr-addendum/f18141e7bf5acfd5ecb90da80261fec3e762bcf7/"
```

File: src/main/data/cplmetadataexts.json
Comment (radford-for-smpte) [inline src/main/data/cplmetadataexts.json:77]:
```suggestion
        "url": "https://docs.dcimovies.com/hdr-addendum/f18141e7bf5acfd5ecb90da80261fec3e762bcf7/"
```

**Comment by SteveLLamb** [comment]
> @SteveLLamb can you clarify if this line is correct in the proposed change:
> 
> https://github.com/msheby/registries/blob/feature/add-hdr1/src/main/data/contentmodifiers.json#L240
> 
> I'm not sure I fully understand how this file is supposed to work.

@radford-for-smpte `scope` in this schema is the defining document of the Content Modifier element or extension, see `Luminance` as an example. Foe extensions, it should be the same as the `ExtScope`, see `Dvis`.

**Comment by radford-for-smpte** [comment]
@SteveLLamb  thanks. Lastly, how do we get the Illustrated Guide updated? Would be nice if `HDR1` was in that top-level box which examples.

**Comment by SteveLLamb** [comment]
> @SteveLLamb thanks. Lastly, how do we get the Illustrated Guide updated? Would be nice if `HDR1` was in that top-level box with examples.
> 
> https://registry-page.isdcf.com/illustratedguide/

Yep, I'll update the code for the SVG separately, once this is merged.

**Review by radford-for-smpte — APPROVED** [review]

**Comment by SteveLLamb** [comment]
Please don't merge this yet. My comments need to be addressed. @msheby I'll push a commit in a few with my suggestions.

**Comment by SteveLLamb** [comment]
> Validation is still failing on the backend with the same `bad API` issue.
You have to merge back in the new master fixes that Pierre pushed to fix that.

**Review by SteveLLamb — APPROVED** [review]

**Comment by SteveLLamb** [comment]
Closes #475

**Pull request merged**
